NSS-Day-Cohort-42/rare-media-moguls
# Cb profile images

#### Changes Made
1. Modified Register.js to allow user to upload profile picture when registering
2. Modified User Details page so that user's profile picture shows up when page is visited
3. Modified route in User Manager so that clicking on email redirects to User Detail page

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-profile-images
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Run with server side branch cb-profile-images, start Django server
> Try to register a new account, and add in your chosen profile image for the account
> Log out then log in as an admin, visit the User Manager page, click on the user's email address to go their User Detail page
> User should have their selected profile image uploaded
5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
